static/data-ingestion.yaml

```
@@ -1235,76 +823,216 @@ paths:
+            type: string
+          description: JWT generated for Production public API key
+        - name: Content-Encoding
+          in: header
+          required: false
+          schema:
+            type: string
+            enum: [ gzip ]
+          description: Use this header if the payload is compressed with gzip
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/FeedPrices'
+            examples:
+              FeedWithProductPricesInformation:
+                summary: Example of product prices feed update
+                description: Updated the feed with product prices information
                 value:
                   [
                     {
-                      "customerGroup": {
-                        "customerGroupCode": "General",
-                        "websites": [
-                          {
-                            "websiteCode": "us"
-                          }
-                        ]
-                      },
-                      "updatedAt": "2023-07-21T21:09:06+00:00",
-                      "deleted": false
+                      "sku": "product-sku-1",
+                      "priceBookId": "price-book-id-1",
+                      "regular": 11.5,
+                      "discounts": [
+                        { "code": "discount", "percentage": 30 }
+                      ]
                     }
                   ]
       responses:
         '200':
-          description: Request accepted. Check the "invalidFeedItems" node for any invalid items that were not consumed
+          description: All items in the request are accepted for further processing
           content:
             application/json;charset=UTF-8:
               schema:
-                $ref: '#/components/schemas/ProcessItemsPublishingResponse'
+                $ref: '#/components/schemas/ProcessFeedResponse'
         '400':
-          description: All feed items received were not valid. Check response body for errors
+          description: Some of the received items are not valid. Check the "invalidFeedItems" node for errors
           content:
             application/json;charset=UTF-8:
               schema:
                 $ref: '#/components/schemas/400ProcessFeedResponse'
         '403':
-          description: Unauthorized request. Validate the "x-api-key" value and ensure the JWT is not expired in "x-gw-signature"
+          description: Unauthorized request. Check the "x-api-key" value and ensure the JWT is not expired in "x-gw-signature"
           content:
             application/json;charset=UTF-8:
               schema:
                 $ref: '#/components/schemas/403Response'
+    delete:
+      tags:
+        - prices
+      summary: Prices Delete API
+      description: >
+        Delete Product Prices API allows you to mark registered product prices as deleted
```

Explain what it means to "mark prices as deleted"  -- what processing happens after delete request?  What if you delete a price that is being used by storefront services?

```suggestion
      summary: Delete prices
      description: >
        Delete existing product prices from the catalog data model.
```

---

Not sure what is meant by the "catalog data" here

---

so from a user point of view --you are deleting a resource from "something" -- what is the something? 

If Data Ingestion is about adding and managing data for an ecommerce product catalog  --